meta-aos-vm-common/recipes-aos/aos-deprov/files/deprovision.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+{
+    sleep 2
+    
+    # use systemctl stop all aos.target services instead of systemctl stop aos.target, because this approach doesn't wait
+    # all services really stopped.
+    systemctl stop -- $(systemctl show -p Wants aos.target | cut -d= -f2)
+
+    #restore unprovisioned flag
+    touch /var/aos/.unprovisioned
```

Comment:
We were going to substitute `.unprovisioned` with `.provisionstate`. So we should remove handling `.unprovisioned` in meta-aos and mea-aos-vm and use .provisiontate insteed